### ISSUE TITLE
Bug: Example in ids_bulk download throws error

### ISSUE THREAD

**Issue by chriscarrollsmith** [Teal-Insights/r-wbids#54]
Teal ran into the following warning and error when running the `ids_bulk` documentation example:

Warning:

The file being downloaded is 128.2 MB in size
There's a 60-second timeout setting that might be insufficient for this large file
The warning suggests using warn_size = FALSE to disable this warning

Error:

The error occurs in the select() function
It's trying to select columns that don't exist
Specifically, the column "Classification Name" doesn't exist in the dataset

The core issue seems to be that after downloading a large data file from the World Bank, the code is trying to select/process columns that aren't present in the downloaded data.

**Comment by chriscarrollsmith** [comment]
Just ran it myself; here's the full output:

```
> data <- ids_bulk(available_files$file_url[1])
  Warning: This file is 128.2 MB and may take several minutes to download. Current timeout
  setting: 60 seconds. Use `warn_size = FALSE` to disable this warning.
  Downloading file to: C:\Users\chris\AppData\Local\Temp\RtmpKyimO9/working_dir\R…
  trying URL 'https://datacatalogfiles.worldbank.org/ddh-published/0038015/DR0092201/A_D.xlsx?versionId=2024-12-04T18:30:10.8890786Z'
  Content type 'application/vnd.openxmlformats-officedocument.spreadsheetml.sheet' length 134381664 bytes (128.2 MB)
  ==================================================
  downloaded 128.2 MB

  Reading in file.
  Processing file.
  Error in `select()`:
  ! Can't select columns that don't exist.
  ✖ Column `Classification Name` doesn't exist.
  Backtrace:
       ▆
    1. ├─(if (getRversion() >= "3.4") withAutoprint else force)(...)
    2. │ └─base::source(...)
    3. │   ├─base::withVisible(eval(ei, envir))
    4. │   └─base::eval(ei, envir)
    5. │     └─base::eval(ei, envir)
    6. ├─wbids::ids_bulk(available_files$file_url[1])
    7. │ └─wbids:::process_bulk_data(bulk_data)
    8. │   ├─tidyr::drop_na(...)
    9. │   ├─dplyr::mutate(...)
   10. │   ├─tidyr::pivot_longer(...)
   11. │   ├─dplyr::select(...)
   12. │   ├─dplyr::select(bulk_raw, -c("Country Name", "Classification Name"))
   13. │   └─dplyr:::select.data.frame(bulk_raw, -c("Country Name", "Classification Name"))
   14. │     └─tidyselect::eval_select(expr(c(...)), data = .data, error_call = error_call)
   15. │       └─tidyselect:::eval_select_impl(...)
   16. │         ├─tidyselect:::with_subscript_errors(...)
   17. │         │ └─base::withCallingHandlers(...)
   18. │         └─tidyselect:::vars_select_eval(...)
   19. │           └─tidyselect:::walk_data_tree(expr, data_mask, context_mask)
   20. │             └─tidyselect:::eval_c(expr, data_mask, context_mask)
   21. │               └─tidyselect:::reduce_sels(node, data_mask, context_mask, init = init)
   22. │                 └─tidyselect:::walk_data_tree(new, data_mask, context_mask)
   23. │                   └─tidyselect:::eval_c(expr, data_mask, context_mask)
   24. │                     └─tidyselect:::reduce_sels(node, data_mask, context_mask, init = init)       
   25. │                       └─tidyselect:::walk_data_tree(new, data_mask, context_mask)
   26. │                         └─tidyselect:::as_indices_sel_impl(...)
   27. │                           └─tidyselect:::as_indices_impl(...)
   28. │                             └─tidyselect:::chr_as_locations(x, vars, call = call, arg = arg)     
   29. │                               └─vctrs::vec_as_location(...)
   30. └─vctrs (local) `<fn>`()
   31.   └─vctrs:::stop_subscript_oob(...)
   32.     └─vctrs:::stop_subscript(...)
   33.       └─rlang::abort(...)
  Execution halted
```

**Comment by christophscheuch** [comment]
Yeah, rather dran *not* selecting columns that we don't need, we should select columns that we need. Also, I realized that the mapping is wrong: we have `counterpart_id = "Series Name"` lol. I'm fixing right now.
